ddionrails/instruments/models/question.py

```diff
@@ -87,6 +87,8 @@ def save(
         self, force_insert=False, force_update=False, using=None, update_fields=None
     ):
         """"Set id and call parents save(). """
+        # pylint: disable=invalid-name
```

Comment:
see line 94

Reply:
black moved the comment
`self.id = hash_with_namespace_uuid(  # pylint: disable=C0103
`
or 
`self.id = hash_with_namespace_uuid(  # pylint: disable=invalid-name`
should work.